src/main/app/controller/PostController.ts

```diff
@@ -83,7 +85,9 @@ export class PostController<T extends AnyObject> {
   }
 
   protected getEventName(req: AppRequest<T>): string {
-    if (req.session.userCase.applicationType === ApplicationType.SOLE_APPLICATION && req.session.isApplicant2) {
+    if (req.session.userCase.state === State.ConditionalOrderDrafted && !req.session.isApplicant2) {
```

Comment:
this is only for a sole application as for a joint app2 can update conditional order

Reply:
yh i was confused on this, tbh i dont know if respondent has any pages to update on this so i left it.

Reply:
ill confirm with BA

Reply:
done